src/5e-SRD-Subclasses.json

```diff
@@ -1,1225 +1,1786 @@
-	"subclass_levels": {
-		"url": "/api/subclasses/evocation/levels"
-	},
-	"url": "/api/subclasses/evocation"
-}]
+[
+  {
+    "index": "berserker",
+    "class": {
+      "index": "barbarian",
+      "name": "Barbarian",
+      "url": "/api/classes/barbarian"
+    },
+    "name": "Berserker",
+    "subclass_flavor": "Primal Path",
+    "desc": [
+      "For some barbarians, rage is a means to an end--that end being violence. The Path of the Berserker is a path of untrammeled fury, slick with blood. As you enter the berserker's rage, you thrill in the chaos of battle, heedless of your own health or well-being."
+    ],
+    "subclass_levels": "/api/subclasses/berserker/levels",
+    "url": "/api/subclasses/berserker"
+  },
+  {
+    "index": "lore",
+    "class": {
+      "index": "bard",
+      "name": "Bard",
+      "url": "/api/classes/bard"
+    },
+    "name": "Lore",
+    "subclass_flavor": "Bard College",
+    "desc": [
+      "Bards of the College of Lore know something about most things, collecting bits of knowledge from sources as diverse as scholarly tomes and peasant tales. Whether singing folk ballads in taverns or elaborate compositions in royal courts, these bards use their gifts to hold audiences spellbound. When the applause dies down, the audience members might find themselves questioning everything they held to be true, from their faith in the priesthood of the local temple to their loyalty to the king. The loyalty of these bards lies in the pursuit of beauty and truth, not in fealty to a monarch or following the tenets of a deity. A noble who keeps such a bard as a herald or advisor knows that the bard would rather be honest than politic. The college's members gather in libraries and sometimes in actual colleges, complete with classrooms and dormitories, to share their lore with one another. They also meet at festivals or affairs of state, where they can expose corruption, unravel lies, and poke fun at self-important figures of authority."
+    ],
+    "subclass_levels": "/api/subclasses/lore/levels",
+    "url": "/api/subclasses/lore"
+  },
+  {
+    "index": "life",
+    "class": {
+      "index": "cleric",
+      "name": "Cleric",
+      "url": "/api/classes/cleric"
+    },
+    "name": "Life",
+    "subclass_flavor": "Divine Domain",
+    "desc": [
+      "The Life domain focuses on the vibrant positive energy--one of the fundamental forces of the universe--that sustains all life. The gods of life promote vitality and health through healing the sick and wounded, caring for those in need, and driving away the forces of death and undeath. Almost any non-evil deity can claim influence over this domain, particularly agricultural deities, sun gods, gods of healing or endurance, and gods of home and community."
+    ],
+    "spells": [
+      {
+        "prerequisites": [
```

Comment:
I was definitely confused by this structure, but I'm not sure of a better way and also outside of scope for this PR.

Reply:
Yeah, this is definitely one for future discussion